backend/server/services/questionService.js

```diff
@@ -1,13 +1,13 @@
 const questionRepository = require('../repositories/questionRepository');
 
-function missingInputsThrowsValidationError(title, complexity, description) {
+const missingInputsThrowsValidationError = (title, complexity, description) => {
   const innerText = description.replace(/<[^>]+>|\s+/g, '');
   if (!title || !complexity || !innerText) {
     throw { status: 400, message: 'Missing inputs' };
   }
 }
```

Comment:
@JrmCkh recommends us to put a `;` at the end of every function.

Reply:
Updated